R/produce_test_comparison.R

```diff
@@ -0,0 +1,15 @@
+#' Produce a test comparison
+#'
+#' @param old_data dataframe containing old file
+#' @param new_data dataframe containing new file
+#'
+#' @return a dataframe with a comparison of new and old data
+#' @family produce tests functions
+produce_test_comparison <- function(old_data, new_data) {
+  dplyr::full_join(old_data, new_data, by = "measure", suffix = c("_old", "_new")) %>%
+    dplyr::mutate(
+      diff = .data$value_new - .data$value_old,
+      pct_change = .data$diff / .data$value_old * 100,
+      issue = if_else(.data$pct_change >= 5, 1, 0)
```

Comment:
Checking for + **and -** changes of > 5%.
The flag can just be TRUE/FALSE.
```suggestion
      issue = !dplyr::between(.data$pct_change, -5, 5)
```